README.md

```diff
@@ -11,6 +11,7 @@ A set of extensions to Doctrine 2 that add support for json query functions.
 | MySQL | `JSON_APPEND, JSON_ARRAY, JSON_ARRAY_APPEND, JSON_ARRAY_INSERT, JSON_CONTAINS, JSON_CONTAINS_PATH, JSON_DEPTH, JSON_EXTRACT, JSON_INSERT, JSON_KEYS, JSON_LENGTH, JSON_MERGE, JSON_MERGE_PATCH, JSON_OBJECT, JSON_QUOTE, JSON_REMOVE, JSON_REPLACE, JSON_SEARCH, JSON_SET, JSON_TYPE, JSON_UNQUOTE, JSON_VALID` |
 | PostgreSQL | `JSON_EXTRACT_PATH, GT, GT_GT, SHARP_GT, SHARP_GT_GT` |
 | MariaDb | `JSON_VALUE, JSON_EXISTS` |
+| SQLite | `JSON, JSON_ARRAY, JSON_ARRAY_LENGTH, JSON_EXTRACT, JSON_GROUP_ARRAY, JSON_GROUP_OBJECT, JSON_INSERT, JSON_OBJECT, JSON_PATCH, JSON_QUOTE, JSON_REMOVE, JSON_REPLACE, JSON_SET, JSON_TYPE, JSON_VALID` |
 
 Table of Contents
 -----------------
```

Comment:
Can you add the new SQLite functions to the Table of Contents in the readme:
```
- [DQL Functions](#dql-functions)
  - [Mysql 5.7+ JSON operators](#mysql-57-json-operators)
  - [PostgreSQL 9.3+ JSON operators](#postgresql-93-json-operators)
  - [SQLite json1 extension operators](#sqlite-json1-extension-operators)
```

Reply:
addint a readme link to https://www.sqlite.org/json1.html below the `[usage]` header would also help for reference